tests/flows/jobs/billing_journal/line_processors/test_base_processor.py

```diff
@@ -0,0 +1,93 @@
+from decimal import Decimal
+from typing import ClassVar
+
+from swo_aws_extension.constants import ItemSkusEnum, UsageMetricTypeEnum
+from swo_aws_extension.flows.jobs.billing_journal.line_processors.base_processor import (
+    GenerateItemJournalLines,
+)
+
+
+class _ValidatorAlwaysTrue:
+    def validate(self, discount, amount, service_name, account_metrics, tolerance_rate):
+        return True
+
+
+class _ValidatorAlwaysFalse:
+    def validate(self, discount, amount, service_name, account_metrics, tolerance_rate):
+        return False
+
+
+class _DummyProcessor(GenerateItemJournalLines):
```

Comment:
Is there a reason for adding the _ prefix in the Dummy / Validator classes? It feels a bit strange